packages/foundations/package.json

```
@@ -20,16 +20,16 @@
     "build:01_normalize": "npm run copy-normalize",
     "build:02_copy": "npm run copy:scss",
     "build:03_css": "sass --no-source-map --load-path=node_modules/ --load-path=../../node_modules/ build/scss/:build/css/ --future-deprecation=import",
-    "build:04_tailwind": "cpr tailwind build/tailwind -o",
+    "build:04_tailwind": "cpy tailwind/** build/tailwind",
     "build:05_postcss": "postcss build/css/**/*.css --replace",
     "clean": "rm -rf build",
     "copy-build": "npm-run-all copy-build:*",
-    "copy-build:assets": "cpr assets ../../build-outputs/foundations/assets -o",
-    "copy-build:package.json": "cpr package.json ../../build-outputs/foundations/package.json -o",
-    "copy-build:readme": "cpr README.md ../../build-outputs/foundations/README.md -o",
-    "copy-build:web": "cpr build ../../build-outputs/foundations/build -o",
-    "copy-normalize": "cpr ../../node_modules/@csstools/normalize.css/normalize.css scss/_normalize.scss -o",
-    "copy:scss": "cpr scss build/scss -o",
+    "copy-build:assets": "cpy assets ../../build-outputs/foundations",
+    "copy-build:package.json": "cpy package.json ../../build-outputs/foundations",
+    "copy-build:readme": "cpy README.md ../../build-outputs/foundations",
+    "copy-build:web": "cpy build ../../build-outputs/foundations",
+    "copy-normalize": "cpy ../../node_modules/@csstools/normalize.css/normalize.css scss --flat --rename=_normalize.scss",
```

```suggestion
    "copy-normalize": "cpy ../../node_modules/@csstools/normalize.css/normalize.css scss --rename=_normalize.scss",
```

It's only one file, does that really need to the `flat` option?

---

Sadly yes. The name of the file and directory is`normalize.css`, causing problems for `cpy`